meta-fobnail-distro/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -1,3 +1,7 @@
 # we promote the usage of chrony instead of the timesyncd - remove so it does
 # not wast space
 PACKAGECONFIG:remove = "timesyncd"
+
+do_install:append() {
```

Comment:
@kotylamichal why we remove those?

Also, workdir/image is destination dir, which variable is `D` https://docs.yoctoproject.org/ref-manual/variables.html#term-D

Reply:
like above, `systemd` creates configs that override our fobnail token configuration. I will add an appropriate comment to this append. `D` macro applied